consent-ui/src/app/ais/sca-select-page/sca-select-page.component.spec.ts

```diff
@@ -35,10 +40,26 @@ describe('ScaSelectPageComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(ScaSelectPageComponent);
     component = fixture.componentInstance;
+    consentAuthorizationService = fixture.debugElement.injector.get(ConsentAuthorizationService);
     fixture.detectChanges();
+    form = component.scaMethodForm;
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should be true if the scaMethodForm is invalid', () => {
+    consentAuthorizationServiceSpy = spyOn(consentAuthorizationService, 'embeddedUsingPOST').and.returnValue(of());
+    form.get('selectedMethodValue').setValue('');
+    component.onSubmit();
+    expect(component.scaMethodForm.invalid).toBe(true);
+  });
+
+  it('should be true if the scaMethodForm is valid', () => {
+    consentAuthorizationServiceSpy = spyOn(consentAuthorizationService, 'embeddedUsingPOST').and.returnValue(of());
+    form.get('selectedMethodValue').setValue(StubUtilTests.SCA_METHOD_VALUE);
+    component.onSubmit();
+    expect(component.scaMethodForm.valid).toBe(true);
```

Comment:
It is good to add expectation that we call backend on submit and (optionally) that we move somewhere based on its response